extension.json

```diff
@@ -16,7 +16,7 @@
 	"descriptionmsg": "wikibase-rdf-description",
 
 	"requires": {
-		"MediaWiki": ">= 1.37.0",
+		"MediaWiki": ">= 1.43.0",
```

Comment:
What are the problems with 1.39? CI was passing years ago, and I don't see any logic changes in this PR. This is mostly CS fixes, method renames and different imports. Don't all the new 1.43 namespaces used in this PR still have old aliases available in 1.43? I don't know if we should drop 1.39 support yet simply because we're using deprecated (but still working) aliases.

Reply:
It isn't a CI issue but code changes upstream with MW and Wikibase between 1.39 and 1.43, especially in database and REST API related classes. We can support 1.39 but it is not worth the time to add the compatibility layer when the 1.X version works fine.